tests/test_main.py

```diff
@@ -284,8 +248,7 @@ def test_initialize_errors(self,
             m.initialize_issues(self.mock_config)
 
         # Assert everything was called correctly
-        mock_u.pagure_issues.assert_called_with('key_pagure', self.mock_config)
-        mock_d.sync_with_jira.assert_any_call('mock_issue_pagure', self.mock_config)
+        mock_u.github_issues.assert_called_with('key_github', self.mock_config)
```

Comment:
You don't want an assertion here like this?
```python
        mock_d.sync_with_jira.assert_any_call('mock_issue_github', self.mock_config)
```

Reply:
Ah yes, my mistake.

Reply:
Handled in [40a3c50](https://github.com/release-engineering/Sync2Jira/pull/208/commits/40a3c505c426a057771a28ec526f832a0e3d663f)